tests/python_tests/ov_genai_test_utils.py

```diff
@@ -0,0 +1,219 @@
+        "meta-llama/Meta-Llama-3-8B-Instruct",
+        "databricks/dbrx-instruct",
+        "mosaicml/mpt-30b-chat",
+        "deepseek-ai/deepseek-coder-6.7b-instruct",
+        "maldv/winter-garden-7b-alpha",
+        "ishorn5/RTLCoder-Deepseek-v1.1",
+        "openchat/openchat-3.5-0106",
+        "casperhansen/llama-3-70b-instruct-awq",
+        "TheBloke/deepseek-coder-33B-instruct-GPTQ",
+        "AI-Sweden-Models/gpt-sw3-356m-instruct",
+        "google/gemma-7b-it",
+        "THUDM/cogvlm2-llama3-chat-19B",
+        "KnutJaegersberg/internlm-20b-llama",
+        "alpindale/WizardLM-2-8x22B",
+        "maywell/Synatra-Mixtral-8x7B",
+        "MediaTek-Research/Breeze-7B-Instruct-v1_0",
+        "bofenghuang/vigostral-7b-chat",
+        "meetkai/functionary-small-v2.5",
+        "nvidia/Llama3-ChatQA-1.5-8B",
+        "openchat/openchat-3.6-8b-20240522",
+        "tenyx/TenyxChat-7B-v1",
+        "LoneStriker/TinyLlama-1.1B-32k-Instruct-3.0bpw-h6-exl2",
+        "yam-peleg/Hebrew-Gemma-11B-V2",
+        "shenzhi-wang/Llama3-8B-Chinese-Chat",
+        "nlpai-lab/KULLM3",
+        "HuggingFaceH4/zephyr-7b-gemma-sft-v0.1",
+        "MediaTek-Research/Breeze-7B-Instruct-v0_1",
+        "shanchen/llama3-8B-slerp-biomed-chat-chinese",
+        "MLP-KTLim/llama-3-Korean-Bllossom-8B",
+        "lucyknada/microsoft_WizardLM-2-7B",
+        "aloobun/CosmicBun-8B",
+        "codellama/CodeLlama-70b-Instruct-hf",
+        "gorilla-llm/gorilla-openfunctions-v2",
+        "BramVanroy/Llama-2-13b-chat-dutch"
+    ]
+    from tokenizer_configs import get_tokenizer_configs
+    return [(k, v) for k, v in get_tokenizer_configs().items() if k not in skipped_models]
+
+
+@functools.lru_cache(1)
+def read_model(params):
+    model_id, path = params
+    
+    from optimum.intel.openvino import OVModelForCausalLM
+    from transformers import AutoTokenizer
+    tokenizer = AutoTokenizer.from_pretrained(model_id, trust_remote_code=True)
+
+    if path.exists():
+        opt_model = OVModelForCausalLM.from_pretrained(path, trust_remote_code=True, 
+                                                       compile=False, device='CPU')
+    else:
+        ov_tokenizer, ov_detokenizer = openvino_tokenizers.convert_tokenizer(tokenizer, 
+                                                                             add_special_tokens=True, 
+                                                                             with_detokenizer=True)
+        openvino.save_model(ov_tokenizer, path / "openvino_tokenizer.xml")
+        openvino.save_model(ov_detokenizer, path / "openvino_detokenizer.xml")
+        
+        # to store tokenizer config jsons with special tokens
+        tokenizer.save_pretrained(path)
+        
+        opt_model = OVModelForCausalLM.from_pretrained(model_id, export=True, trust_remote_code=True, 
+                                                       compile=False, device='CPU', load_in_8bit=False)
```

Comment:
I just moved this piece without changes. If model was already `saved_pretrained` no need to download it again. I suspect that even if we always do `OVModelForCausalLM.from_pretrained(model_id..` it will not download but use model form cache but still I see no harm in the existing code. Can we keep it?